src/Features/Core/Portable/EditAndContinue/Remote/RemoteDebuggingSessionProxy.cs

```diff
@@ -88,25 +89,26 @@ await client.TryInvokeAsync<IRemoteEditAndContinueService>(
             return new EmitSolutionUpdateResults.Data()
             {
                 ModuleUpdates = new ModuleUpdates(ModuleUpdateStatus.RestartRequired, []),
-                Diagnostics = GetInternalErrorDiagnosticData(solution, e),
+                Diagnostics = GetInternalErrorDiagnosticData(e.Message),
                 RudeEdits = [],
                 SyntaxError = null,
                 ProjectsToRebuild = [],
                 ProjectsToRestart = [],
             };
         }
-    }
 
-    private static ImmutableArray<DiagnosticData> GetInternalErrorDiagnosticData(Solution solution, Exception e)
-    {
-        var descriptor = EditAndContinueDiagnosticDescriptors.GetDescriptor(RudeEditKind.InternalError);
+        ImmutableArray<DiagnosticData> GetInternalErrorDiagnosticData(string message)
+        {
+            var descriptor = EditAndContinueDiagnosticDescriptors.GetDescriptor(RudeEditKind.InternalError);
 
-        var diagnostic = Diagnostic.Create(
-            descriptor,
-            Location.None,
-            string.Format(descriptor.MessageFormat.ToString(), "", e.Message));
+            var firstProject = solution.GetProject(runningProjects.FirstOrDefault()) ?? solution.Projects.First();
```

Comment:
so for EnC/HotReload, i took this approach to assign these generic "something went wrong" to the first project being run.  Note: i couldn't tell if runningProjects could be empty or not.  If not, i can simplify this.

Reply:
Let's keep it as is for now.